### PR TITLE
Add note about Jaeger Agent hostname for sidecar deployments

### LINKED ISSUE #299: Document sidecar agent hostname
## Requirement - what kind of business use case are you trying to solve?
We need to know what host name to use when configuring Jaeger Client libraries. (In my case that's [opencensus-ruby-exporter-jaeger](https://github.com/Thinkei/opencensus-ruby-exporter-jaeger))

## Problem - what in Jaeger blocks you from solving the requirement?
I didn't find anything in the documentation about injecting sidecars that explained what host to connect to.

I eventually guessed it when I read the [jaeger-client-java source](https://github.com/jaegertracing/jaeger-client-java/blob/6548879da36eb26d712037866b90685d1209ff1f/jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/UdpSender.java#L27). Then I confirmed it by running this little script in one of my pods:
```
# test.sh
if timeout 5 bash -c "</dev/$1/$2/$3 &>/dev/null"
then
echo "Port is open"
else
echo "Port is closed"
fi
# end of file
chmod +x

$ ./test.sh udp localhost 6831
udp
Port is open
```
## Proposal - what do you suggest to solve the problem or improve the existing situation?
I'm preparing a PR for the documentation for v1.13, v1.14 and next release because I'm running 1.13

## Any open questions to address
None

### PR DESCRIPTION
**Pull request by danielmbarlow**
## Which problem is this PR solving?
Resolves #299 

## Short description of the changes
Added note about jaeger sidecar agent hostname to v1.13, v.14 and next release

### CODE REVIEW

File: content/docs/1.13/operator.md [hunk 1]
```diff
@@ -421,6 +421,8 @@ spec:
 
 A complete sample deployment is available at [`deploy/examples/business-application-injected-sidecar.yaml`](https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/examples/business-application-injected-sidecar.yaml).
 
+When the sidecar is injected, the Jaeger Agent can then be accessed on `localhost`.
```
Comment (objectiser) [inline content/docs/1.13/operator.md]: Maybe change "accessed on" to "at its default location on"? As by default the jaeger clients will assume the agent is running on localhost.
Same change to be applied in 1.14/next-release.

File: content/docs/1.14/operator.md [hunk 2]
```diff
@@ -413,6 +413,7 @@ The operator can inject Jaeger Agent sidecars in `Deployment` workloads, provide
 
 The following snippet shows a simple application that will get a sidecar injected, with the Jaeger Agent pointing to the single Jaeger instance available in the same namespace:
 
+
```
Comment (objectiser) [inline content/docs/1.14/operator.md]: Could you remove this extra line.

**Comment by objectiser** [comment]
@danielmbarlow each commit needs to be signed - probably easiest way now is to just squash the last two commits.

**Comment by danielmbarlow** [comment]
Sorry @objectiser I forgot to sign the last two. I squashed all of them into one.

**Pull request merged**